raiden/blockchain_events_handler.py

```diff
@@ -91,67 +92,65 @@ def handle_channel_new(raiden: "RaidenService", event: Event):
     channel_identifier = args["channel_identifier"]
     participant1 = args["participant1"]
     participant2 = args["participant2"]
-    is_participant = raiden.address in (participant1, participant2)
+
 
     # Check if at least one of the implied participants is a LC handled by the node
-    is_participant1_handled_lc = LightClientService.is_handled_lc(to_checksum_address(encode_hex(participant1)),
-                                                                  raiden.wal)
-    is_participant2_handled_lc = LightClientService.is_handled_lc(to_checksum_address(encode_hex(participant2)),
-                                                                  raiden.wal)
+    is_participant1_handled_lc = LightClientService.is_handled_lc(
```

Comment:
so, we've talked about this method before (specifically, the issue of it being too lengthy).

having taken another look at it now, i can see that most of the lines are just arguments for `create_channel`. i think this would be best solved by using a class for all the parameters (there's way too many of them in my opinion) but i don't see this as a blocking issue.

so, in summary, i would leave as is for now